README.md

```diff
@@ -811,6 +812,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [minio-java](https://github.com/minio/minio-java) - Provides simple APIs to access any Amazon S3 compatible object storage server.
 * [Protégé](http://protege.stanford.edu/) - Provides an ontology editor and a framework to build knowledge-based systems.
 * [Underscore-java](https://github.com/javadev/underscore-java) - Port of Underscore.js functions.
+* [strman-java](https://github.com/shekhargulati/strman-java): A Java 8 library for working with String.
```

Comment:
@shekhargulati Thanks!

I may have sounded a bit loaded earlier, so if I did sound loaded, apologies!

I realise I wasn't terribly clear: it's not that authors aren't allowed to submit their own projects, it's just they will be reviewed more critically than otherwise, since self-promotion is frowned upon a bit - but if they meet the criteria, they'll be accepted. The [CONTRIBUTING.md](https://github.com/akullpp/awesome-java/blob/master/CONTRIBUTING.md) goes into more detail about this. :)